examples/native-hybrid-app/README.md

```diff
@@ -0,0 +1,53 @@
+# "Add-To-App" or Hybrid Native + Flutter Applciation Example
+
+This example covers how to use the Datadog Flutter SDK in conjunction with an already existing native application. This assumes you already have a native iOS or Android Application that is sending data to Datadog, and you have added Flutter to it following Flutter's [Add-to-app documentation](https://docs.flutter.dev/development/add-to-app).
+
+## Native App is Primary and Datadog is Already Initialized -  `attachToExisting`
+
+If you are using an application that is already using the native Datadog iOS or Datadog Android SDKs, the Flutter SDK can attach to these using the same parameters. In your `main` function, after calling `WidgetsFlutterBinding.ensureInitialized`, call `DatadogSdk.instance.attachToExisting`. You can optionally add a `LoggingConfiguraiton` to this call, which automatically creates a global logger and attaches it to `DatadogSdk.logs`.
+
+```dart
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  final config = DdSdkExistingConfiguration(
+    loggingConfiguration: LoggingConfiguration()
+  );
+
+  await DatadogSdk.instance.attachToExisting(config);
+
+  runApp(MyApp());
+}
+```
+
+Additional options for `DdSdkExistingConfiguration` are documented in the [API reference](https://pub.dev/documentation/datadog_flutter_plugin/latest/datadog_flutter_plugin/datadog_flutter_plugin-library.html). Note that options in this class need to be re-specified even if they are already passed during Native SDK initialization.
+
+### Avoiding "FlutterViewController" and "FlutterActivity" views
+
+Depending on your settings, the automatic view tracking in the native iOS and Android SDKs will automatically track the presentation of the `FlutterViewController` and `FlutterActivity`/`FlutterFragment` when they appear, and then immediately add show a view load for your tracked Flutter view.
```

Comment:
```suggestion
Depending on your settings, the automatic view tracking in the native iOS and Android SDKs will automatically track the presentation of the `FlutterViewController` and `FlutterActivity`/`FlutterFragment` when they appear, and then immediately add a view for your tracked Flutter view.
```
This looks to me a like an edit pass left a few extra words by mistake. I'm not sure what the intended phrase was, so I used my best guess.